manifests/common/apps/jupyter/create-jupyter-notebook-quickstart.yaml

```diff
@@ -22,26 +22,26 @@ spec:
       description: |-
         ### Prerequisites
         - You have logged in to Open Data Hub.
-        - You know the names and values that you want to use for environment variables in your notebook server environment, for example, `AWS_SECRET_ACCESS_KEY`.
-        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your notebook server.
+        - You know the names and values that you want to use for environment variables in your workbench environment, for example, `AWS_SECRET_ACCESS_KEY`.
+        - If you want to work with a very large data set, work with your administrator to proactively increase the storage capacity of your workbench.
         ### Launch a notebook
         1. On the **Enabled applications** page, locate the **Jupyter** card.
```

Comment:
Are we changing the **Jupyter** card to "Start basic workbench" card in ODH too? Based on @kaedward's doc we'll be changing it for RHOAI, but not sure if that is the same for ODH.